### PR TITLE
[APB-3386][MP] Moving params for client type page into config

### PR DESCRIPTION
**Pull request by mariannepearson0**

### CODE REVIEW

File: app/uk/gov/hmrc/agentinvitationsfrontend/views/agents/ClientTypePageConfig.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentinvitationsfrontend.views.agents
+
+import play.api.i18n.Messages
+
+case class ClientTypePageConfig(backLinkUrl: Option[String])(implicit messages: Messages) {
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentinvitationsfrontend/views/agents/ClientTypePageConfig.scala]: do we need this case class at all ? 
we can use clientTypes straightaway in the views, with out needing to pass from the controllers (?)

Comment (mariannepearson0) [reply app/uk/gov/hmrc/agentinvitationsfrontend/views/agents/ClientTypePageConfig.scala]: clientTypes is now in the view config for that page, not being passed by the controller...

Comment (mariannepearson0) [reply app/uk/gov/hmrc/agentinvitationsfrontend/views/agents/ClientTypePageConfig.scala]: Just trying to keep vals and functions where they are used and using the config to extract as much logic out of the html as possible

Comment (sureshhmrc) [reply app/uk/gov/hmrc/agentinvitationsfrontend/views/agents/ClientTypePageConfig.scala]: ok, not a big thing, lets merge

**Pull request merged**